src/couch_replicator_scheduler.erl

```diff
@@ -41,7 +41,7 @@
 %% definitions
 -define(MAX_BACKOFF_EXPONENT, 10).
 -define(BACKOFF_INTERVAL_MICROS, 30 * 1000 * 1000).
-
+-define(HEALTH_THRESHOLD_MICROS, 2 * 60 * 1000 * 1000).
```

Comment:
2 minutes was picked thinking about the amount of time (even with long socket timeouts) for a job to have a chance to start working and fail. 

Initially wanted to make it equal to `interval`. But then saw it should have a lower bound, as users could set scheduler interval to 5 seconds, then that would be too low. 

Currently if we look at the `stopped` event, the minimum would be 1 interval, but it would be a pathological case. For example, users would have to configure `max_churn = max_jobs` and there should be `max_churn` worth of `pending_jobs()` available to run. Normally a job will not be stopped after each interval. In general it should be after`(max_jobs/max_churn) * interval` so by default (500/20) * 1 min = 25min

Also in the code, it would be a bit messy as would have to pass in the Interval or config value from state through 4 functions (`pending_jobs/1`, `ets:foldl(...pending_fold/2..)`, `pending_fold/2`, `not_recently_crashed`...).